apps/cms-server/modules/openstad-auth/index.js

```diff
@@ -101,7 +132,7 @@ module.exports = {
               });
             };
 
-            const ONE_MINUTE = 60 * 1000;
+            const ONE_MINUTE = 1 * 1000;
```

Comment:
Dit is nu niet meer ONE_MINUTE maar eerder ONE_SECOND nu toch? Waarom is dit aangepast?

Reply:
Beter inderdaad. Een minuut is te lang om te kijken of iemand nog is ingelogd. Ik heb er nu 5 dsec van gemaakt; ff kijken hoe dat gaat.